src/rfc3161_client/tsp.py

```diff
@@ -166,7 +166,7 @@ def nonce(self) -> bytes:
 
     @property
     @abc.abstractmethod
-    def name(self) -> cryptography.x509.Name:
+    def name(self) -> cryptography.x509.GeneralName:
```

Comment:
FLAG: The annotation was wrong but I didn't think we had to create a whole new PR just for it :)